models/netsuite2/netsuite2__balance_sheet.sql

```diff
@@ -129,132 +153,133 @@ balance_sheet as (
 
     {% endif %}
 
-  from transactions_with_converted_amounts
-  
-  --Below is only used if balance sheet transaction detail columns are specified dbt_project.yml file.
```

Comment:
@fivetran-catfritz Will we still have the ability to pass through columns from transaction detail? I may have missed it somewhere.